ultralytics/cfg/datasets/carparts-seg.yaml

```diff
@@ -0,0 +1,35 @@
+path: "../datasets/carparts-seg" # dataset root dir
+train: train/images  # training images directory
+val: valid/images    # validation images directory
+test: test/images   # testing images directory
+
+nc: 23    # number of classes
+
+# name of classes
+names:
+  0: back_bumper
+  1: back_door,
```

Comment:
@RizwanMunawar I think there's an extra comma here?

Reply:
Wait but @RizwanMunawar the comma is still here. Please remove it if it shouldn't be there.

Reply:
Ok I've fixed it now.